service/controller/release/resource/status/cluster.go

```diff
@@ -197,9 +146,7 @@ func (r *Resource) listPartialObjectMetadata(ctx context.Context, gvk metav1.Gro
 			}.String(),
 		},
 	}
-	err := r.k8sClient.CtrlClient().List(ctx, &results, &client.ListOptions{
```

Comment:
Try to find clusters on all namespaces because we use the organization namespace now.